pkgs/applications/science/physics/crystfel/default.nix

```
@@ -0,0 +1,133 @@
+{ lib, stdenv, fetchurl, gfortran, m4, hdf5, gsl, unzip, makeWrapper, meson, git, ninja, eigen, pkg-config, wrapGAppsHook, flex, bison, doxygen, opencl-headers, ncurses, msgpack, fftw, zeromq, ocl-icd, gtk3, gdk-pixbuf }:
```

```suggestion
{ lib, stdenv, fetchurl, gfortran, m4, hdf5, gsl, unzip, makeWrapper, meson, git, ninja, eigen, pkg-config
, wrapGAppsHook, flex, bison, doxygen, opencl-headers, ncurses, msgpack, fftw, zeromq, ocl-icd, gtk3, gdk-pixbuf }:
```
long lines are okay but they should fit approximately onto onto the github page without scrolling.